<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grippersimulator</class>
 <widget class="QWidget" name="grippersimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="Alignment_Info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="distanceSlider">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>200</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="angleSlider">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>270</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="distanceLCD">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="angleLCD">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="distanceLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Target_Distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="angleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Target_Angle</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="targetAlignment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>targetAlignment</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Grabber_Aligned">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Grabber_Aligned</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
